addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -585,6 +587,27 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * @private
+         * @param {Object} data
+         */
+        _handleNotificationPartnerIap(data) {
+            if (data.is_credit_error) {
+                const content = `<a class='btn btn-link' href=${data.url} target='_blank' ><i class='fa fa-arrow-right'/>${this.env._t("Buy more credits")}</a>`;
+                this.env.services['notification'].displayNotification({
+                    title: data.title,
+                    message: content,
+                    messageIsHtml: true,
+                });
+            } else {
+                this.env.services['notification'].displayNotification({
+                    title: this.env._t("Success"),
+                    message: data.title,
+                    type: 'success',
+                });
+            }
+        }
+
```

Comment:
```suggestion
        _handleNotificationPartnerIap(data) {
            let notification;
            if (data.is_credit_error) {
                const content = `<a class='btn btn-link' href=${data.url} target='_blank' ><i class='fa fa-arrow-right'/>${this.env._t("Buy more credits")}</a>`;
                notification = {
                     title: data.title,
                     message: content,
                     messageIsHtml: true,
                };
            } else {
                notification = {
                    title: this.env._t("Success"),
                    message: data.title,
                    type: 'success',
                };
            }
          this.env.services['notification'].displayNotification(notification);
        }
```
What do you think (notwithstanding the poor formatting)